### PR TITLE
Section header fix (#465)

### PR DESCRIPTION
**Pull request by VegetarianZombie**
This pull request resolves the issue with additional lines rendering in iOS 14. This was due to list sections being nested inside of each other. This pull request refactors the sections inside of a group that resolves the issue. It also sets the width of the list cell to the width of the device.
 
Before:

Screen Shot 2020-09-14 at 2 53 04 PM

After:

Screen Shot 2020-09-14 at 2 41 32 PM

Note: The project contains a Group struct that causes a name collision with the SwiftUI Group.

### CODE REVIEW

**Review by sammyd — COMMENTED** [review]
This looks good to me. Since it requires Xcode 12, I want to wait until we can switch CI to merge it. Hopefully it'll be in the next day or so.

**Comment by sammyd** [comment]
@VegetarianZombie actually, how come the styling of the header has changed? I don't think it should be centred / capitalised.

**Comment by VegetarianZombie** [comment]
Hi Sam - Okay, that was a mistake on my part. I'll update the PR to use the correct styling information.

File: Emitron/Emitron/UI/Shared/Content Detail/CourseHeaderView.swift
Comment (sammyd) [inline Emitron/Emitron/UI/Shared/Content Detail/CourseHeaderView.swift]: Not sure why, but this is now centred and capitals?

**Review by sammyd — CHANGES_REQUESTED** [review]
As specified above—not sure why the typography has changed...

**Comment by VegetarianZombie** [comment]
Hi @sammydl, I investigated the issue today. The casing change is a result from iOS 14. By default, section headers appear now to be uppercased. This can be changed with the 'textCase' modifier.

I updated the pull request to left justify the header. I also added a conditional statement to set the text case on iOS 14 and above. Pre iOS 14 is fine.

**Comment by sammyd** [comment]
Thanks @VegetarianZombie. I'll get this rebased, resolve the merge conflicts, and get it merged.

**Review by sammyd — APPROVED** [review]
:+1: I've rebased it, and will go ahead and merge it once CI has completed.

Nice work—thanks for this @VegetarianZombie

**Comment by sammyd** [comment]
This fixed #465

**Pull request merged**
